community/rds/aurora-serverless/template.yml

```diff
@@ -0,0 +1,70 @@
+---
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: Aurora serverless cluster
+
+Parameters:
+  DatabaseName:
+    Type: String
+  MasterUserPassword:
+    Type: AWS::SSM::Parameter::Value<String>
+    Description: Parameter Store name
+  VpcId:
```

Comment:
VpcId parameters should use the AWS-Specific "AWS::EC2::VPC::Id" parameter type (see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html)